hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/schemas/InitialModServiceTokenSchema.java

```diff
@@ -104,10 +104,10 @@ public class InitialModServiceTokenSchema extends Schema {
     // These need to be big so databases are created at right scale. If they are too small then the on disk hash map
     // buckets will be too full which results in very poor performance. Have chosen 10 billion as should give us
     // plenty of runway.
-    private static final long MAX_TOKENS = 10_000_000_000L;
-    private static final long MAX_ACCOUNTS = 10_000_000_000L;
-    private static final long MAX_TOKEN_RELS = 10_000_000_000L;
-    private static final long MAX_MINTABLE_NFTS = 10_000_000_000L;
+    private static final long MAX_TOKENS = 1_000_000_000L;
```

Comment:
Comment above needs fixing `10 billion` (and mention that 10B was _too_ large because someone in the future may find a reason to want to up this?)

Reply:
Right, will fix this in a follow-up PR.